packages/base64/test/bench-main.js

```diff
@@ -0,0 +1,46 @@
+import { encodeBase64, decodeBase64 } from '../src/main';
+
+const string = new Array(100)
+  .fill(
+    'there once a rich man from nottingham who tried to cross the river. what a dope, he tripped on a rope. now look at him shiver.',
```

Comment:
Where is this from?

Reply:
It’s a song Will Scarlett uses to mock Kevin Costner’s Robin Hood.

Reply:
Resolving ;)